sos/report/plugins/ovirt.py

```diff
@@ -87,11 +88,39 @@ def setup(self):
             self.add_forbidden_path('/var/log/ovirt-engine/dump')
             self.add_cmd_output('ls -l /var/log/ovirt-engine/dump/')
 
+        certificates = [
+            '/etc/pki/ovirt-engine/ca.pem',
+            '/etc/pki/ovirt-engine/apache-ca.pem',
+            '/etc/pki/ovirt-engine/certs/engine.cer',
+            '/etc/pki/ovirt-engine/certs/apache.cer',
+            '/etc/pki/ovirt-engine/certs/websocket-proxy.cer',
+            '/etc/pki/ovirt-engine/certs/jboss.cer',
+            '/etc/pki/ovirt-engine/certs/imageio-proxy.cer',
+            '/etc/pki/ovirt-engine/certs/ovirt-provider-ovn.cer',
+
+        ]
+
+        keystores = [
+            ('mypass', '/etc/pki/ovirt-engine/.truststore'),
+            ('changeit', '/var/lib/ovirt-engine/external_truststore'),
+        ]
+
         self.add_cmd_output([
             # Copy all engine tunables and domain information
             "engine-config --all",
             # clearer diff from factory defaults (only on ovirt>=4.2.8)
-            "engine-config -d"
+            "engine-config -d",
+        ])
+
+        self.add_cmd_output([
+            # process certificate files
+            "openssl x509 -in %s -text -noout" % c for c in certificates
+        ])
```

Comment:
No need to wrap the command in a list here - `add_cmd_output()` accepts either a list of strings or a single string as the command argument.